modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceShipmentLocalServiceImpl.java

```diff
@@ -65,6 +66,55 @@
 public class CommerceShipmentLocalServiceImpl
 	extends CommerceShipmentLocalServiceBaseImpl {
 
+	@Indexable(type = IndexableType.REINDEX)
+	@Override
+	public CommerceShipment addCommerceDeliverySubscriptionShipment(
```

Comment:
We should name this,
```suggestion
	public CommerceShipment addDeliverySubscriptionCommerceShipment(
```